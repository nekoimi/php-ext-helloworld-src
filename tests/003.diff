001+ Fatal error: Uncaught Error: Call to undefined function helloworld_test2() in /home/nekoimi/developer/cpp-project/php-src/ext/helloworld/tests/003.php:2
002+ Stack trace:
001- string(11) "Hello World"
002- string(9) "Hello PHP"
003+ #0 {main}
004+   thrown in /home/nekoimi/developer/cpp-project/php-src/ext/helloworld/tests/003.php on line 2