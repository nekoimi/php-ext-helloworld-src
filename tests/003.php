<?php
var_dump(helloworld_test2());
var_dump(helloworld_test2('PHP'));
?>
