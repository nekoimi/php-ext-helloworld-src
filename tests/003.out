Fatal error: Uncaught Error: Call to undefined function helloworld_test2() in /home/nekoimi/developer/cpp-project/php-src/ext/helloworld/tests/003.php:2
Stack trace:
#0 {main}
  thrown in /home/nekoimi/developer/cpp-project/php-src/ext/helloworld/tests/003.php on line 2